jsonnet/kube-thanos/kube-thanos-rule.libsonnet

```diff
@@ -183,4 +183,42 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       },
     },
   },
+
+  withAlertmanagers:: {
+    local tr = self,
+    config+:: {
+      ruleConfigMapName: error 'must provide rulesConfigMapName',
```

Comment:
we need to be consistent here; it should either be `rulesConfigMapName` or `ruleConfigMapName` without the `s`

Reply:
forgot to update the message.